src/fparser/statements.py

```diff
@@ -666,11 +679,22 @@ def process_item(self):
         return
 
     def tofortran(self, isfix=None):
-        t = ''
+        '''
+        Create the Fortran for this ALLOCATE statement
```

Comment:
Perhaps "Create the Fortran code..." or "Create the Fortran representation..."?

Reply:
Done.